scipy-1.0/paper.tex

```diff
@@ -122,6 +122,21 @@ \section*{Project organisation and community}
 
 \textbf{Governance}
 
+SciPy adopted an official Governance Document on August 3, 2017 (see PR\#6953). In summary,
+we have a Benevolant Dictator for Life (BDFL) in Pauli Virtanen, and a (currently) 18
+member Steering Council chaired by Ralf Gommers that i.e., decides on who gets commit rights.
+
+SciPy adopted an official Code of Conduct on October 24, 2017 (see PR\#7963), drawing
+inspiration from the Apache Foundation Code of Conduct, the Contributor Covenant,
+the Jupyter Code of Conduct, and the Open Source Guides - Code of Conduct. In short,
+we stated five specific guidelines: \emph{be open} (to anyone participating in our community),
```

Comment:
Should we use the exact wording? Some of these are a little different from the actual document.